test/src/test/resources/rmq-proxy-home/conf/logback_proxy.xml

```diff
@@ -81,10 +81,10 @@
 
     <appender name="RocketmqBrokerAppender_inner"
               class="ch.qos.logback.core.rolling.RollingFileAppender">
-        <file>${user.home}/logs/rocketmqlogs/${brokerLogDir}/broker.log</file>
+        <file>${user.home}/logs/rocketmqlogs/${brokerLogDir:-${file.separator}}/broker.log</file>
         <append>true</append>
         <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
-            <fileNamePattern>${user.home}/logs/rocketmqlogs/otherdays/${brokerLogDir}/broker.%i.log.gz</fileNamePattern>
+            <fileNamePattern>${user.home}/logs/rocketmqlogs/otherdays/${brokerLogDir:-${file.separator}}/broker.%i.log.gz</fileNamePattern>
```

Comment:
ditto.

Reply:
Done, and confirm there is any /logs exists in logback config xml